tests/ui/chalkify/bugs/async.rs

```diff
@@ -1,6 +1,7 @@
 // edition:2021
 // known-bug: unknown
 // unset-rustc-env:RUST_BACKTRACE
+// rustc-env:RUSTC_ICE_LOG=trace
```

Comment:
I wanted to see one that actually captured some SpanTrace output just to make sure it's working and i understood why the other ones are empty.

I'm going to add some logic to skip adding the "SpanTrace:\n" bit if the SpanTrace itself is empty. I'm also wondering if I should add some `cfg` directives to set the `RUSTC_ICE_LOG` level to trace when we're not building rustc itself but I need to investigate how that would work.

Reply:
This test suddenly started failing after rebasing on master and I do not yet know why